GitPrompt.ps1

```diff
@@ -138,13 +138,22 @@ function Write-GitStatus($status) {
             }
             $repoName = Split-Path -Leaf (Split-Path $status.GitDir)
             $prefix = if ($s.EnableWindowTitle -is [string]) { $s.EnableWindowTitle } else { '' }
-            $Host.UI.RawUI.WindowTitle = "$prefix$repoName [$($status.Branch)]"
+            $adminHeader = Get-AdminTitleHeader

```

Comment:
Probably both statements are true.  The cost is trivial compared to every time you spin up a new process to call git.exe, but there's also no reason _not_ to cache the admin status, since it's never going to change within the same session.